changes/1275-kesavkolla.md

```diff
@@ -0,0 +1 @@
+Made user info optional for `RedisDSN`
```

Comment:
If we're adding `password_required` and `credentials_required` we should put that in the change description.